siege_utilities/political/__init__.py

```
@@ -0,0 +1,53 @@
+            tracking_table="_schema_migrations",
+        )
+        political_runner.apply_all()
+
+        # Then run the consumer's own migrations.
+    """
+    return Path(__file__).resolve().parent / "migrations"
+
+
+from siege_utilities.political import readers  # noqa: F401
+
+__all__ = ["schema_migrations_dir", "readers"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)